shuup/front/basket/command_dispatcher.py

```diff
@@ -73,8 +74,9 @@ def handle(self, command, kwargs=None):
         except (Problem, ValidationError) as exc:
             if not self.ajax:
                 raise
+            msg = exc.message if hasattr(exc, "message") else exc
```

Comment:
As discussed in Slack. It's OK to use `exc.message` in this case. So this change is OK.